pkg/apis/operator.min.io/v1/types.go

```diff
@@ -39,6 +42,22 @@ type MinIOInstance struct {
 	Status    MinIOInstanceStatus    `json:"status"`
 }
 
+// Validate returns an error if any configuration of the MinIO instance is invalid
+func (in *MinIOInstance) Validate() error {
+	// Make sure the storage request is not 0
+	if in.Spec.VolumeClaimTemplate.Spec.Resources.Requests.Storage().Value() == 0 {
+		return errors.New("cannot use volume size 0")
```

Comment:
Add details for the claim name and other details to help debug which instance.
The operator could be spinning up more than one instance and will help search for instance specific logs.

Reply:
Since we have a single `VolumeClaimTemplate` on the MinIOInstance we don't need to specify down to the `pvc` level, we are preventing the statefulset even being created by stopping the minioinstance at this point